File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.js
// Last updated: 01/04/2026, 23:34:33
1/**
2 * @param {number[]} nums
3 * @return {number}
4 */
5var removeDuplicates = function(nums) {
6   if(nums.length===0)
7   return 0;
8   let i=0;
9   let j=0;
10   let len= nums.length;
11   while(j<len){
12    if(nums[j]!==nums[i]){
13        i++;
14        nums[i]=nums[j];
15    }
16    j++;
17   }
18   return i+1;
19};

File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.cpp
// Last updated: 13/04/2026, 15:30:24
1class Solution {
2public:
3    int removeDuplicates(vector<int>& nums) {
4        if (nums.size() == 0) return 0;
5
6        int i = 0;
7
8        for (int j = 1; j < nums.size(); j++) {
9            if (nums[j] != nums[i]) {
10                i++;
11                nums[i] = nums[j];
12            }
13        }
14
15       return i+1;
16    }
17};